examples/walkthrough/README.md

```diff
@@ -1,74 +1,167 @@
+
+```sh
+kubectl create -f examples/walkthrough/pod-nginx.yaml
+```
+
+List all pods:
+
+```sh
+kubectl get pods
+```
+
+Provided your pod IPs are accessible (depending on network setup), you should be able to access the pod's http endpoint in a browser or curl on port 80.
+
+View the pod's http endpoint:
+
+```sh
+curl http://$(kubectl get pod nginx -o=template -t={{.status.podIP}})
+```
+
+Delete the pod by name:
+
+```sh
+kubectl delete pod nginx
+```
+
+
+#### Volumes
+
+That's great for a simple static web server, but what about persistent storage?
+
+The container file system only lives as long as the container does. So if your app's state needs to survive relocation, reboots, and crashes, you'll need to configure some persistent storage.
+
+For this example, we'll be creating a Redis pod, with a named volume and volume mount that defines the path to mount the volume.
+
+1. Define a volume:
+
+  ```
+    volumes:
+    - name: redis-persistent-storage
+      emptyDir: {}
+  ```
+
+1. Define a volume mount within a container definition:
+
+```
+    volumeMounts:
+    # name must match the volume name below
+    - name: redis-persistent-storage
+      # mount path within the container
+      mountPath: /data/redis
+```
+
+Example Redis pod definition with a persistent storage volume ([pod-redis.yaml](pod-redis.yaml)):
+
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: storage
+  name: redis
 spec:
   containers:
-    - name: redis
-      image: redis
-      volumeMounts:
-          # name must match the volume name below
-        - name: redis-persistent-storage
-          # mount path within the container
-          mountPath: /data/redis
-  volumes:
+  - name: redis
+    image: redis
+    volumeMounts:
     - name: redis-persistent-storage
-      emptyDir: {}
-```
-
-Ok, so what did we do? We added a volume to our pod:
-```
+      mountPath: /data/redis
   volumes:
-    - name: redis-persistent-storage
-      emptyDir: {}
-```
-
-And we added a reference to that volume to our container:
-```
-      volumeMounts:
-          # name must match the volume name below
-        - name: redis-persistent-storage
-          # mount path within the container
-          mountPath: /data/redis
+  - name: redis-persistent-storage
+    emptyDir: {}
 ```
 
-In Kubernetes, ```emptyDir``` Volumes live for the lifespan of the Pod, which is longer than the lifespan of any one container, so if the container fails and is restarted, our persistent storage will live on.
+In Kubernetes, ```emptyDir``` Volumes live for the lifespan of the Pod, which may be longer than the lifespan of any one container, so if the container fails and is restarted, our persistent storage will live on.
 
 If you want to mount a directory that already exists in the file system (e.g. ```/var/logs```) you can use the ```hostDir``` directive.
 
 See [volumes](../../docs/volumes.md) for more details.
 
-### Multiple Containers
+
+#### Multiple Containers
 
 _Note:
```

Comment:
It means that nobody has built and pushed the required container image for the example, so the example doesn't work exactly as written.